mcs/class/corlib/Test/System/DecimalTest.cs

```diff
@@ -882,6 +882,8 @@ public void TestRound ()
 				{1.234567890m, 2, 1.23m}, {1.23450000001m, 3, 1.235m}, 
 				{1.2355m, 3, 1.236m}, 
 				{1.234567890m, 4, 1.2346m}, {1.23567890m, 2, 1.24m}, 
+				{1.00567890m, 2, 1.01m},
+				{1.005m, 2, 1.01m},
```

Comment:
Yeah, I agree this is confusing.

Let me point you to this excellent [SO answer](http://stackoverflow.com/questions/2226081/why-does-net-use-a-rounding-algorithm-in-string-format-that-is-inconsistent-wit) by Eric Lippert:
> Perhaps this is yet another case of 25 years of backwards compatibility.

:smile: